src/client/canvas/svg/text.tsx

```diff
@@ -0,0 +1,37 @@
+import { observer } from 'mobx-react'
+import * as React from 'react'
+import { Component } from 'react'
+
+import { Transform } from '../../math/transform'
+import { TextGeometry } from '../geometry/text_geometry'
+import { Shape } from '../object/shape'
+
+import { appearance, transform } from './svg'
+
+@observer
+export class Text extends Component<{model: Shape<TextGeometry>, world: Transform}> {
+
+  public render() {
+    // TODO handle font size and max width properly
+
+    const m = this.props.model
```

Comment:
Can we just restructuring instead of 1 letter variable names? Here is how I would write this class:

```ts
export class Text extends Component<{ model: Shape<TextGeometry>, world: Transform }> {
  public render() {
    const { world, model: { geometry, appearance } } = this.props
    const transform = Transform.of(geometry.alignToView ? {
      scale: { x: Math.sign(world.scale.x), y: Math.sign(world.scale.y) },
      rotate: world.rotate,
    } : undefined)

    return (
      <text
        x={geometry.x}
        y={geometry.y}
        fontFamily={geometry.fontFamily}
        fontSize={'0.015rem'} // TODO handle font size and max width properly
        textAnchor={geometry.textAlign}
        dominantBaseline={geometry.textBaseline}
        transform={toSvgTransform(transform)}
        {...toSvgAppearance(appearance)}
      >
        {geometry.text}
      </text>
    )
  }
}
```

If you agree, please make sure all the others are also consistent 🙂